src/client/components/common/CodeSnippet/CodeSnippet.tsx

```diff
@@ -0,0 +1,106 @@
+import { CSSProperties, PropsWithChildren } from 'react';
+import { PrismAsyncLight } from 'react-syntax-highlighter';
+import lightTheme from 'react-syntax-highlighter/dist/cjs/styles/prism/coy';
+import styles from './CodeSnippet.module.scss';
+import classnames from 'classnames';
+import { MyScrollArea } from '../ScrollArea/ScrollArea';
+import { MyCopyButton } from '../CopyButton/CopyButton';
+import JsonView from 'react18-json-view';
+import 'react18-json-view/src/style.css';
+import { CopyButtonSvg } from '../../../img/CopyButtonSvg';
+import { CheckMarkSvg } from '../../../img/CheckmarkSvg';
+
+export interface CodeSnippetProps {
+  language: string;
+  children: string;
+  showCopyButton?: boolean;
+  showLineNumbers?: boolean;
+  className?: string;
+  dataTestId?: string;
+  collapsibleJSON?: boolean;
+}
+
+const PRISM_LINE_NUMBER_STYLE = { minWidth: 28 };
+
+/**
+ * Provides a syntax-highlighted code block
+ */
+export function CodeSnippet({
+  language,
+  showLineNumbers,
+  className,
+  children,
+  dataTestId,
+}: PropsWithChildren<CodeSnippetProps>) {
+  const PRISM_CUSTOM_STYLE: CSSProperties = {
+    padding: '16px',
+    fontFamily: 'JetBrains Mono, monospace',
+    fontSize: '16px',
+    overflow: 'auto',
+    marginTop: '0',
+    marginBottom: '0',
+    marginLeft: '0',
+    marginRight: '0',
+  };
+  const PRISM_CODE_TAG_PROPS = { style: { color: '#c92c2c', font: 'inherit' as const } };
```

Comment:
nit: Let's move this constants outside of the component

Reply:
Thanks, fixed: https://github.com/fingerprintjs/fingerprintjs-pro-use-cases/commit/92cd7afa8ed5538543133f02f780c9bd90e0686e